podcasts/End of Year/EndOfYearStoriesBuilder.swift

```diff
@@ -95,6 +96,14 @@ class EndOfYearStoriesBuilder {
                 stories.append(.longestEpisode)
             }
 
+            // Year over year listening time
+            let yearOverYearListeningTime = dataManager.yearOverYearListeningTime()
+            if yearOverYearListeningTime.totalPlayedTimeThisYear != 0 ||
+                yearOverYearListeningTime.totalPlayedTimeLastYear != 0 {
```

Comment:
We can't compare if we don't have history for both years, so we should hide the story. 

```suggestion
            if yearOverYearListeningTime.totalPlayedTimeThisYear != 0 &&
                yearOverYearListeningTime.totalPlayedTimeLastYear != 0 {
```

Reply:
We can't compare but we'll show it, we'll display the previous year as "0 minutes listened". ([discussed here](https://a8c.slack.com/archives/C041BGTFQ5R/p1697573573229879))